SECP-service/src/main/java/com/visucius/secp/auth/TokenAuthFilter.java

```diff
@@ -23,7 +23,7 @@
 
     @Override
     public void filter(ContainerRequestContext containerRequestContext) throws IOException {
-        final String authToken = containerRequestContext.getHeaders().getFirst("token");
+        final String authToken = containerRequestContext.getHeaders().getFirst("Authorization");
 
         if (StringUtils.isBlank(authToken)) {
```

Comment:
Wow so it was authorization here as opposed to token.

Reply:
yep :)